deploy/06_secondary_scheduler_cr.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: operator.openshift.io/v1
```

Comment:
hmm, it should show up in the `alm_examples` of the generated csv, may be the difference in namespace?